examples/TensorFlow/Inference/BERT/README.md

```diff
@@ -0,0 +1,36 @@
+# DISC optimization example for TensorFlow inference.
+
+This repository provides a script showing how to optimize a BERT large inference model with DISC just-in-time.
```

Comment:
is it possible to put the test datas in OSS, and put everything in a single file?
So that we can export to HTMLs.
Refer to examples/pytorch/inference/torch_bert_tutorial.md